### DISCUSSION TITLE
NEED HELP !!!!!! How can i get a new suscribe?I use this for the first time.The help seems useless!

### DISCUSSION THREAD

**Discussion by Nfzwy-del** [community/community#131761] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### i can"t find the website for subscribe!  The help is useless
<img width="761" alt="截屏2024-07-11 上午11 12 31" src="https://github.com/community/community/assets/175275925/9dbc91a4-072f-4011-8ce2-7fbdee38f666">

<img width="779" alt="截屏2024-07-11 上午11 12 26" src="https://github.com/community/community/assets/175275925/2799e302-76b6-41d3-acb0-f92579b5774e">

<img width="831" alt="截屏2024-07-11 上午11 12 18" src="https://github.com/community/community/assets/175275925/95b1a04e-abbe-456e-b0c6-40148128f8be">

look at these photos!
when i start the clash and click the CLASH META nothing happened and a note come out please look at the photo below!

<img width="422" alt="截屏2024-07-11 上午11 16 04" src="https://github.com/community/community/assets/175275925/fd6651bc-5ad4-4cc5-8334-0b4ac40b3cad">

beside that i can't also find the right website for add a new suscribe

i hope someone can help me!
i just use this platform for the fist time.
PLEASE GIVE ME SOME SUPPORT!
THANK YOU!
I NEED HELP!

**Comment by LiteBrite82** [comment, marked as answer]
Hi @Nfzwy-del ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this software. 

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
